ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -19,6 +19,8 @@ let prototyped_of_field = function
       Some "22.27.0"
   | "host", "last_software_update" ->
       Some "22.20.0"
+  | "VM", "actions__after_softreboot" ->
```

Comment:
Because of all the releases done (sorry!) this change needs to be deleted and make rerun to regenerate the file